src/Perl6/Grammar.nqp

```diff
@@ -4348,6 +4349,22 @@ grammar Perl6::Grammar is HLL::Grammar does STD {
         <O(|%conditional, :reducecheck<ternary>, :pasttype<if>)>
     }
 
+    token infix:sym<⁇ ‼> {
```

Comment:
yes, just copy-pasted. I don't know if there is a better solution... any suggestion?